en/simulation/multi-vehicle-simulation.md

```diff
@@ -19,14 +19,13 @@ To test an example setup, follow the below steps,
   ```
   cd Firmware_clone
   git submodule update --init --recursive
-  make posix_sitl_default gazebo
+  make posix_sitl_default sitl_gazebo
```

Comment:
Does this (using `sitl_gazebo`) just build the code without running it? (Just interested, normally I only see `make posix_sitl_default gazebo_"target"`

Reply:
What if we want to launch a different model - e.g. tailsitter?

Reply:
`make posix_sitl_default gazebo` makes the `sitl_gazebo` submodule, but it also launch the gazebo viewer. We don't want to open the gui yet. That is what the launch file is for.

Reply:
If you want to use a different vehicle, you need a xacro that represents the model and you need to specify the `vehicle` arg in the launch to reflect this.

I guess this is confusing because in this case, when using ROS, we don't want make to end up starting sitl and opening things. We just want it to build the firmware and sitl app.